pravega-provisioner/cluster_provisioner.py

```diff
@@ -38,6 +39,30 @@ def get_int_input(msg, valid_values):
     return user_input
 
 
+def get_int_input_with_default(msg, valid_values, default_value):
```

Comment:
I don't see this method being used. If so, please remove it.

Reply:
done